Source: @pullmoll
Upstream: no
Reason: expression requires binary AND, not logical

--- a/libsrc/util/pcrelib/pcre_study.c	2018-08-16 01:06:21.000000000 +0200
+++ b/libsrc/util/pcrelib/pcre_study.c	2020-01-27 15:44:57.715821355 +0100
@@ -432,7 +432,7 @@
           for (c = 0; c < 16; c++) start_bits[c] |= tcode[c];
           for (c = 128; c < 256; c++)
             {
-            if ((tcode[c/8] && (1 << (c&7))) != 0)
+            if ((tcode[c/8] & (1 << (c&7))) != 0)
               {
               int d = (c >> 6) | 0xc0;            /* Set bit for this starter */
               start_bits[d/8] |= (1 << (d&7));    /* and then skip on to the */
